apps/emqx_audit/src/emqx_audit.erl

```diff
@@ -50,7 +50,13 @@ to_audit(#{from := cli, cmd := Cmd, args := Args, duration_ms := DurationMs}) ->
         http_method = <<"">>,
         http_request = <<"">>
     };
-to_audit(#{from := From} = Log) when From =:= dashboard orelse From =:= rest_api ->
+to_audit(#{from := From} = Log) when
```

Comment:
in case someone forgets to add a new From here in the future,
should we add a default clause to emit a warning log instead of function_clause ?